src/app/app-modules/lab/workarea/workarea.component.ts

```
@@ -965,16 +965,14 @@ export class WorkareaComponent
-          const url = window.URL.createObjectURL(blob);
-          const a = document.createElement('a');
-          a.href = url;
-          a.download = result.fileName;
-          document.body.appendChild(a);
-          a.click();
-          document.body.removeChild(a);
+        const fileID = {
+          fileID: result,
+        };
+        this.labService.viewFileContent(fileID).subscribe((res: any) => {
+          if (res && res.data && res.data.statusCode === 200) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)